src/components/manage-productions/manage-productions.tsx

```diff
@@ -24,6 +24,7 @@ const SubContainers = styled.div`
   width: 100%;
   display: flex;
   flex-direction: row;
+  margin-bottom: 2rem;
```

Comment:
the commit `fix: added padding to confirm-remove message ` does not seem to add any padding?